package.json

```diff
@@ -56,12 +56,12 @@
     "eslint-plugin-header": "^3.1.1",
     "eslint-plugin-prettier": "^4.0.0",
     "express": "^4.17.1",
-    "grpc-tools": "^1.11.1",
     "grpc_tools_node_protoc_ts": "^5.3.0",
+    "grpc-tools": "^1.11.1",
     "ioredis": "^4.27.2",
     "jest": "^26.6.3",
     "lint-staged": "^12.3.2",
-    "mongodb": "^3.6.4",
+    "mongodb": "^4.5.0",
```

Comment:
I am not sure whether this is a good practise.

@tom-pytel @kezhenxu94 How nodeJS agent controls the versions of plugin supporting?

Reply:
it's in the devDependencies, take not effect to the application